### ISSUE TITLE
Invalid link in readme

### ISSUE THREAD

**Issue by ChillerDragon** [thelinuxchoice/instashell#36]
This link from the bottom at the readme does not work.
https://github.com/antony-jr/instagram-py

Maybe this is the repo?
https://github.com/deathsec/instagram-py

**Comment by silver0321** [comment]
@ChillerDragon Yep that's the right link but the original author deleted the project because it does not work anymore. The second repo is just a fake user who copied the project and re-uploaded. You can actually see the git history(**Which is now deleted**) and see who coded everything.

See: https://github.com/antony-jr/ama/issues/36#issuecomment-456699704

You can see that the official python package on instagram-py uses all reference to the first link, take a look here, https://pypi.org/project/instagram-py/

So I think the author used the right url but the source is deleted so you have 404 not found.
